libraries/helpers.rb

```diff
@@ -32,7 +32,7 @@ def vagrant_platform_package(vers = nil)
     case node['platform_family']
     when 'debian'
       "vagrant_#{vers}_x86_64.deb"
-    when 'redhat', 'fedora'
```

Comment:
Right, didn't see that you were not checking `node['os']`. I think that it would be a good change.